deliveries/typechain_revived_milestone_2.md

```diff
@@ -0,0 +1,27 @@
+| 0b.    | Documentation                           | [Documentation](https://github.com/C-Forge/typechain-polkadot/blob/main/docs/about)                              | Additionally each packages contains its own extensive README.md.                                                                                                                                                                                               |
+| 0c.    | Testing and Testing Guide               | [Tests & Testing Guide](#https://github.com/C-Forge/typechain-polkadot/tree/main/tests)                          |                                                                                                                                                                                                                                                                |
+| 0d.    | Article                                 | [Article](#https://github.com/C-Forge/typechain-polkadot/blob/main/docs/whats_new.md)                            | Covers improvements, reasonings behind them, as well as overall DX benefits.                                                                                                                                                                                   |
+| **1**  | SC Deployment code                      | [Changeset](#https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | We will have simplified & enhanced usage of deployment classes. Code's API will be consulted with the community to ensure it fits its needs.                                                                                                                   |
+| **2**  | Improve Result/Error handling           | [Changeset](#https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | Fixed several bugs & implemented correct error decoding                                                                                                                                                                                                        |
+| **3**  | Handle bugs & issues                    | [Fixed in](#https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)       | Addressed all the issues and bugs we encountered while developing typechain & contributing to Abax Protocol/Governance. Additionally, we fixed all feature related issues from Brushfam's issue tracker https://github.com/Brushfam/typechain-polkadot/issues. |
+| **4**  | Incorporate Docker build into Typechain | [Implemented in](#https://github.com/C-Forge/typechain-polkadot/commit/f15c78f224d2dbdaa3af770a4f13af1316bd5e59) |                                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| 0c.    | Testing and Testing Guide               | [Tests & Testing Guide](https://github.com/C-Forge/typechain-polkadot/tree/main/tests)                          |                                                                                                                                                                                                                                                                |
| 0d.    | Article                                 | [Article](https://github.com/C-Forge/typechain-polkadot/blob/main/docs/whats_new.md)                            | Covers improvements, reasonings behind them, as well as overall DX benefits.                                                                                                                                                                                   |
| **1**  | SC Deployment code                      | [Changeset](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | We will have simplified & enhanced usage of deployment classes. Code's API will be consulted with the community to ensure it fits its needs.                                                                                                                   |
| **2**  | Improve Result/Error handling           | [Changeset](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | Fixed several bugs & implemented correct error decoding                                                                                                                                                                                                        |
| **3**  | Handle bugs & issues                    | [Fixed in](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)       | Addressed all the issues and bugs we encountered while developing typechain & contributing to Abax Protocol/Governance. Additionally, we fixed all feature related issues from Brushfam's issue tracker https://github.com/Brushfam/typechain-polkadot/issues. |
| **4**  | Incorporate Docker build into Typechain | [Implemented in](https://github.com/C-Forge/typechain-polkadot/commit/f15c78f224d2dbdaa3af770a4f13af1316bd5e59) |                                                                                                                                                                                                                                                                |
```

Fixing hyperlinks

Reply:
Thanks @semuelle and sorry about those typos.